docs/source/development/automated_testing.md

```diff
@@ -21,7 +21,7 @@ Let's look at how you can start working with `pytest` in your Kedro project.
 
 ### Install `pytest`
 
-Install `pytest` as you would install other packages with `pip`, making sure your project's virtual environment is active.  If you're unfamiliar with virtual environments, see our [docs page on the subject](https://kedro.readthedocs.io/en/stable/get_started/prerequisites.html#virtual-environments) for a quick primer.
+Install `pytest` as you would install other packages with `pip`, making sure your project's virtual environment is active.  If you're unfamiliar with virtual environments, see our [docs page on the subject](https://kedro.readthedocs.io/en/stable/get_started/install.html#virtual-environment-manager) for a quick primer.
```

Comment:
```suggestion
Install `pytest` as you would install other packages with `pip`, making sure your project's virtual environment is active.  If you're unfamiliar with virtual environments, see our [docs page on the subject](../get_started/install.md#virtual-environment-manager) for a quick primer.
```

Reply:
We shouldn't hard code RTD links into our docs. Please use relative links and the docs build tools will pick this up before the publication process.